### PR TITLE
Initial Job setup

### PR DESCRIPTION
**Pull request by todorivanov**

### CODE REVIEW

File: src/Lykke.AlgoStore.Job.GDPR.Services/SubscriberService.cs [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+                        });
+                }
+
+                // find and stop all instances
+                var instances = await _instanceRepository.GetAllAlgoInstancesByClientAsync(clientId);
+
+                foreach (var instance in instances)
+                {
+                    var result =
+                        await _algoInstanceStoppingClient.DeleteAlgoInstanceAsync(instance.InstanceId,
```
Comment (todorivanov) [inline src/Lykke.AlgoStore.Job.GDPR.Services/SubscriberService.cs:175]: Should we change the "ClientId" here?

File: src/Lykke.AlgoStore.Job.GDPR.Services/SubscriberService.cs [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+using System.ComponentModel.DataAnnotations;
+using System.Threading.Tasks;
+using Common.Log;
+using Lykke.Common.Log;
+
+namespace Lykke.AlgoStore.Job.GDPR.Services
+{
+    public class SubscriberService : ISubscriberService
+    {
+        private readonly ISubscriberRepository _usersRepository;
```
Comment (todorivanov) [inline src/Lykke.AlgoStore.Job.GDPR.Services/SubscriberService.cs]: rename the repository

File: src/Lykke.AlgoStore.Job.GDPR/Controllers/SubscribersController.cs [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+{
+    [Route("api/v1/subscribers")]
+    public class SubscribersController: Controller
+    {
+        private readonly ISubscriberService _usersService;
+        private readonly ILog _log;
+
+        public SubscribersController(ISubscriberService usersService, ILogFactory logFactory)
+        {
+            _usersService = usersService;
```
Comment (todorivanov) [inline src/Lykke.AlgoStore.Job.GDPR/Controllers/SubscribersController.cs]: Rename service

**Review by ghost — APPROVED** [review]

File: src/Lykke.AlgoStore.Job.GDPR.Services/SubscriberService.cs (hunk 1 above)
Comment (dafinkagerdjikova) [reply src/Lykke.AlgoStore.Job.GDPR.Services/SubscriberService.cs:175]: It's updated in the repo

File: src/Lykke.AlgoStore.Job.GDPR/Controllers/SubscribersController.cs (hunk 3 above)
Comment (dafinkagerdjikova) [reply src/Lykke.AlgoStore.Job.GDPR/Controllers/SubscribersController.cs]: I will update it

**Review by iliyan-ivanov-prime — APPROVED** [review]

**Pull request merged**